_i18n/en.yml

```diff
@@ -475,18 +475,32 @@ faq:
 mining:
   intro1: Monero is a cryptocurrency that relies on proof-of-work mining to achieve distributed consensus. Below you'll find some information and resources on how to begin mining.
   intro2: The Monero Project does not endorse any particular pool, software, or hardware, and the content below is provided for informational purposes only.
-  support: Support
-  support_para1: See
-  support_para2: Hangouts
-  support_para4: and
-  pools: Pools
-  pools_para1: A listing of Monero pools is found
-  pools_para2: here.
-  benchmarking: Hardware Benchmarking
-  benchmarking_para1: See here
-  benchmarking_para2: for a listing of GPUs/CPUs and their respective hashrates.
-  software: Mining Software
+  pow: Proof of Work
+  philosophy: >
+    One of Monero's philosophies is to maintain egalitarian mining, so that everyone can have the possibility to mine. To achieve this, Monero uses a particular algorithm ideated and developed by members of the Monero community: @RandomX. This PoW algorithm is ASIC resistant, which means it's impossible to build specialized hardware to mine Monero. Miners must use consumer-grade hardware and compete fairly.
```

Comment:
```suggestion
    One of Monero's philosophies is to maintain egalitarian mining, so that everyone can have the possibility to mine. To achieve this, Monero uses a particular algorithm ideated and developed by members of the Monero community: @RandomX. This PoW algorithm is ASIC resistant, which means specialized hardware has less of an advantage at mining Monero. Miners can use consumer-grade hardware and compete fairly.
```

Reply:
```
19:56 <sech1> "which means it's impossible to build specialized hardware to mine Monero"
19:56 <sech1> not impossible
20:07 <selsta> "This PoW algorithm is ASIC resistant, which means specialized hardware has less of an advantage at mining Monero. Miners can use consumer-grade hardware and compete fairly"
20:07 <selsta> better ?
20:11 <hyc> sounds good
20:11 <sech1> better, but it's hard to explain all nuances in a single sentence
20:12 <moneromooo> This better. Us use it.
20:12 <moneromooo> That's already two sentences :/
20:12 <hyc> lol
20:13 <hyc> I was thinking "less benefit" but less advantage is good enough
20:14 <moneromooo> Maybe "playing on the strengths of modern CPU architectures mean it is much harder for specialized hardware to get an edge".
20:14 <hyc> at this point in time, ASIC advantage <= 0
20:15 <hyc> ^ the above is a teeny tiny bit better than "This better. Us use it."
```

Suggestions from IRC.

Reply:
I would go for something like

```
which means is not economically convenient to build specialized hardware to mine Monero
```
I think it makes more clear that ASICs are not in the network, even if they can exist.

Reply:
I personally prefer my suggestion but you can decide. If you use your suggestion don't forget to add an `it`.

Reply:
I think my version leaves less space to misunderstanding. I would go for

```
which means it'snot economically convenient to build specialized hardware (ASIC) to mine Monero
```